d3a_api_client/commands.py

```diff
@@ -47,22 +47,21 @@ def device_info(self, area_uuid):
         return self._add_to_buffer(area_uuid, Commands.DEVICE_INFO, {})
 
     def last_market_stats(self, area_uuid):
-        return self._add_to_buffer(area_uuid, "market_stats", {"data": {}})
+        return self._add_to_buffer(area_uuid, Commands.MARKET_STATS, {"data": {}})
 
     def last_market_dso_stats(self, area_uuid):
-        return self._add_to_buffer(area_uuid, "dso_market_stats", {"data": {}})
+        return self._add_to_buffer(area_uuid, Commands.DSO_MARKET_STATS, {"data": {}})
 
     def change_grid_fees_percent(self, area_uuid, fee_percent):
-        return self._add_to_buffer(area_uuid, "grid_fees", {"data": {"fee_percent": fee_percent}})
+        return self._add_to_buffer(area_uuid, Commands.GRID_FEES, {"data": {"fee_percent": fee_percent}})
 
     def grid_fees(self, area_uuid, fee_cents_kwh):
-        return self._add_to_buffer(area_uuid, "grid_fees", {"data": {"fee_const": fee_cents_kwh}})
+        return self._add_to_buffer(area_uuid, Commands.GRID_FEES, {"data": {"fee_const": fee_cents_kwh}})
 
     def _add_to_buffer(self, area_uuid, action, args):
         if area_uuid and action:
             self._commands_buffer.append(
-                {area_uuid: {"type": command_enum_to_command_name(action)
-                if type(action) == Commands else action, **args}})
```

Comment:
I think this extra precaution could stay. Of course, it is only an extra safety-net for the programmer, but still. Thanks!